packages/monitors/index.js

```diff
@@ -324,10 +327,10 @@ async function run({
     if (optimisticOracleAddress) {
       const optimisticOracleContractEventClient = new OptimisticOracleEventClient(
         logger,
-        getAbi("OptimisticOracle"),
+        getAbi(optimisticOracleType),
```

Comment:
would it be useful to validate that this is of an appropriate type? it would be possible to provide a type that breaks the bot here (i.e a string that is not the correct type)

Reply:
We already do this by making sure that `optimisticOracleType` is of type `OptimisticOracleType`. This is in javascript but if converted to typescript it would validate for free